src/Development/IDE/Spans/LocalBindings.hs

```diff
@@ -45,13 +53,42 @@ bindings refmap = Bindings $ L.foldl' (flip (uncurry IM.insert)) mempty  $ do
        , unitNameEnv name (name,ty)
        )
 
+------------------------------------------------------------------------------
+-- | Compute which identifiers are in scope at every point in the AST. Use
+-- 'getDefiningBindings' to find the results.
+bindSites
+    :: RefMap
+    -> IntervalMap Position (NameEnv (Name, Maybe Type))
```

Comment:
I think it would be good to combine this with the local bindings traversal. If not, at least use `foldl'` instead of `foldr`

Reply:
Yeah probablly. I didn't have the mental energy to figure out how to navigate the do block for the two cases though. Will think about it further tomorrow.